client/components/library-items.vue

```diff
@@ -98,9 +117,12 @@
 <template>
     <section id="libraryContainer">
         <h2>Gear</h2>
-        <input id="librarySearch" v-model="searchText" type="text" placeholder="search items">
+        <div id="librarySearch">
+            <input v-model="searchText" type="text" placeholder="search items"/>
+            <p v-if="searchText.length > 0" @click="clearSearch">&times;</p>
```

Comment:
could use native clear by changing the input from type text to search
```suggestion
            <input v-model="searchText" type="search" placeholder="search items"/>
```
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search